README.md

```diff
@@ -67,6 +67,16 @@ of NIST-standardized elliptic curves and symmetric cryptosystems.
 [Stanford Pairing-Based Crypto (PBC) library](http://crypto.stanford.edu/pbc/),
 which will of course only work if the PBC library is installed.
 
+Issues
+------
+
+- Traditionally, ECDH uses only the x point for the shared secret. In this
```

Comment:
This text is very unclear and out-of-the-blue; it needs some context, and in particular it's not obvious what exactly "uses" means here.  Can you expand the term "ECDH", and point to the most-relevant document specifying ECDH's standard behavior.  And you need to clarify here that "uses" means "forms the Diffie-Hellman shared secret from..."